docs/BEMAN_PROCESS.md

```diff
@@ -0,0 +1,53 @@
+<!--
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+# The Beman Process
+
+This document is an extension of [BEMAN_STANDARD.md](./BEMAN_STANDARD.md)
+and contains additional best practices and guidelines related to Beman development.
+Its goal is to create consistency facilitating the evaluation of, and contribution to Beman libraries.
+
+Examples: Beman libraries lifetime; integration with Compiler Explorer; how to write Beman docs etc.
+
+## Introduction
+
+### Changing this document
+
+This is a living document that adapts to evolving best practices and community
+needs. To make changes:
+
+1. Create a [discourse topic](https://discourse.bemanproject.org/latest) detailing the changes
+and how it aligns with the core principles.
+2. After some community discussion, create a PR with the actual changes on
+[GitHub](https://github.com/bemanproject/beman) with the `Beman Leads`
+label. The PR should also link to the discourse topic.
+3. Continue discussions on the PR and discourse topic.
+4. Await a leads a decision based on the community feedback.
```

Comment:
Agree. This was just copy paste from BEMAN_STANDARD. So my proposal is to first incorporate your feedback in existing section from BEMAN_STANDARD.md#Changing-this-document.

And then, I would actually say that this section would be more apropiate in the root README.md, because in my opinion, any doc in `beman/` repo will need discussion for important changes. (root README or `docs/README.md`).

@JeffGarland @bemanproject/leads ?